src/handlers/bops/v2/publicComments.ts

```diff
@@ -59,6 +62,23 @@ export async function publicComments(
     if (searchParams.orderBy) {
       params.append("orderBy", searchParams.orderBy);
     }
+    if (searchParams.sentiment) {
+      params.append("sentiment", searchParams.sentiment);
+    }
+    if (searchParams.publishedAtFrom) {
+      params.append("publishedAtFrom", searchParams.publishedAtFrom);
+      if (searchParams.publishedAtTo === "") {
+        params.append("publishedAtTo", new Date().toISOString().split("T")[0]);
+      }
+    }
+    if (searchParams.publishedAtTo) {
+      params.append("publishedAtTo", searchParams.publishedAtTo);
+      if (searchParams.publishedAtFrom === "") {
+        const date = new Date(searchParams.publishedAtTo);
+        date.setDate(date.getDate() - 1);
+        params.append("publishedAtFrom", date.toISOString().split("T")[0]);
+      }
+    }
```

Comment:
We need to update the `publishedAtTo` and `publishedAtFrom` fields in the URL after someone clicks search so users can see what date range was used in the request.

Since we're working with plain old html, the logic for setting the fallback values for those fields can be moved into `fetchCommentData` in `src/app/[council]/[reference]/comments/page.tsx`. That way the query string passed to the server reflects what was actually queried.